<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="dataBaseFileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Database File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="minimunStrenghLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimun Block Strengh </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="minimunStrenghInputText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="maximunStrenghLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximun Block Strengh </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="maximunStrenghInputText">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="noClusterRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>No clustering</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="existingClusterRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing implied clusters</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="existingWeightsRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>201</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Existing sequence weights</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="PositionBasesWeigthsRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Position-Based Weights, PB Weight = 1/n</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="clusterPercentageRadioButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>310</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Cluster Percentage</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="clusterPercentageInputText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clusterOptionLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Cluster Option</string>
    </property>
   </widget>
   <widget class="QLabel" name="Scalelabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale n For 1/n Bits</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>BLOSUM MATRIX</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="scaleInputText">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>370</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>450</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateMatrixButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Matrix</string>
    </property>
   </widget>
   <widget class="QLabel" name="fileNameLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>481</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="positionBasedWeightsInputText">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>620</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
